scaper/core.py

```diff
@@ -1,4 +1,7 @@
-import sox
+try:
+    import soxbindings as sox 
+except: # pragma: no cover
+    import sox # pragma: no cover
```

Comment:
I think this is good for now, but let's create an issue so we remember that in the long term this needs to be addressed. It looks like GH actions supports running tests on all 3 OSs, so in an ideal scenario we can switch to GH actions and run the tests on all platforms, including Windows.

<img width="934" alt="Screen Shot 2020-09-23 at 9 49 32 AM" src="https://user-images.githubusercontent.com/3009670/94044064-82304a00-fd82-11ea-8ce7-8e0af4ab55b6.png">